WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -121,9 +121,11 @@ public enum WooAnalyticsStat: String {
     case sitePickerStoresShown = "site_picker_stores_shown"
     case sitePickerHelpButtonTapped = "site_picker_help_button_tapped"
     case sitePickerNonWooSiteTapped = "site_picker_non_woo_site_tapped"
-    case sitePickerEnterStoreAddressTapped = "site_picker_enter_store_address_tapped"
     case sitePickerSiteDiscovery = "site_picker_site_discovery"
     case sitePickerNewToWooTapped = "site_picker_new_to_woo_tapped"
+    case sitePickerAddStoreTapped = "site_picker_add_a_store_tapped"
+    case sitePickerCreateNewStoreTapped = "site_picker_create_store_tapped"
```

Comment:
this event is also part of the analytics plan in pe5sF9-I3-p2#comment-1147 - `site_picker_create_site_tapped` with a `source` property, which I'm implementing today (PR soon). Maybe you can leave this one out for now?

Reply:
(It's also logged in Android)

Reply:
Sure, I'll remove my event. Thanks for letting me know!